handlers/schemas/UploadVOD.yaml

```
@@ -105,6 +105,13 @@ properties:
       - "width"
       -  "height"
       - "bitrate"
+  encryption:
+    type: "object"
+    properties:
+      encryptedKey: string
+    required: ["encryptedKey"]
```

Catalyst API is in camel_case
```suggestion
      encrypted_key: string
    required: ["encrypted_key"]
```